java_gen/pre-written/src/main/java/org/projectfloodlight/openflow/types/OFChecksum128.java

```diff
@@ -0,0 +1,80 @@
+package org.projectfloodlight.openflow.types;
+
+import io.netty.buffer.ByteBuf;
+
+import com.google.common.hash.PrimitiveSink;
+
+public class OFChecksum128 implements OFValueType<OFChecksum128> {
```

Comment:
Is this intended to be here / related to the netty upgrade? I'm asking because I don't see this being referred to in this pull request or the corresponding pull requests downstream.